website/screens/components/date-input/code/DateInputCodePage.tsx

```diff
@@ -196,13 +198,27 @@ const sections = [
         ),
       },
       {
-        title: "Error management",
+        title: "Error handling",
         content: (
           <>
-            <DxcText as="p">
-              Example of error management when it does not match the format.
-            </DxcText>
-            <Example example={errorManagement} defaultIsVisible />
+            <DxcText as="p">Example of error handling when it does not match the format.</DxcText>
```

Comment:
We should explain a bit more how error handling works, since it does more than it seems. Look at [Radio Group's example](https://developer.dxc.com/tools/react/next/#/components/radioGroup) for Error handling.